shell/platform/windows/fixtures/main.dart

```diff
@@ -127,6 +127,21 @@ void exitTestCancel() async {
   await exited.future;
 }
 
+@pragma('vm:entry-point')
+void enableLifecycleTest() async {
+  final Completer<ByteData?> finished = Completer<ByteData?>();
+  ui.channelBuffers.setListener('flutter/lifecycle', (ByteData? data, ui.PlatformMessageResponseCallback callback) async {
+    if (data != null) {
+      ui.PlatformDispatcher.instance.sendPlatformMessage(
+        'flutter/unittest',
+        data,
+        (ByteData? reply) {
+        });
+    }
+  });
+  await finished.future;
```

Comment:
It looks like you're never completing this future. If that's intentional, add a comment as to why.

Reply:
It might actually be useful to describe the back and forth being done over the channels to verify the handling/enabling in a comment in the C++ side of the unit tests, then maybe a one-liner here pointing readers to that test.